src/components/Input.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+
+interface InputProps {
+    onClick?: () => void;
+    className?: string;
+    title: string;
+  }
+
+const Input=({
+    className = '',
+    title,
+  }: InputProps)=> {
+  const [isChecked, setIsChecked] = useState(false);
+
+  return (
+    <>
+      <div className="flex gap-2">
+        <input
+          type="checkbox"
+          checked={isChecked}
+          onChange={() => setIsChecked((prev) => !prev)}
```

Comment:
For scalability, could you please create the function to handle this onChange.

Reply:
Have made the changes. Will keep this in mind from next time.